ui/js/reducers/file_info.js

```diff
@@ -18,7 +18,9 @@ reducers[types.FILE_LIST_COMPLETED] = function(state, action) {
 
   const newFileInfos = Object.assign({}, state.fileInfos)
   fileInfos.forEach((fileInfo) => {
-    newFileInfos[fileInfo.outpoint] = fileInfo
+    const { outpoint } = fileInfo
+
+    if (outpoint) newFileInfos[fileInfo.outpoint] = fileInfo
```

Comment:
I have a file with no outpoint locally. It's one that I tried to upload but it didn't work for some reason.